src/main/java/com/libraryman_api/borrowing/BorrowingRepository.java

```diff
@@ -11,6 +11,19 @@
 @Repository
 public interface BorrowingRepository extends JpaRepository<Borrowings, Integer> {
 
+public Page<BorrowingsDto> getAllBorrowingsOfMember(int memberId, Pageable pageable) {
```

Comment:
It seems the `getAllBorrowingsOfMember` method was added by mistake. Could you please re-add the `findByMember_memberId` method signature as it was previously?
 

```java
Page<Borrowings> findByMember_memberId(int memberId, Pageable pageable);
```